resolwe/process/models.py

```diff
@@ -268,12 +266,84 @@ class Model(metaclass=ModelMetaclass):
 
     _app_name = "App name"
     _model_name = "Model name"
+    # A default list of extra fields that are returned on filter call.
+    _filter_response_fields = []
 
     def __init__(self, pk: int):
         """Initialization."""
         self._pk = pk
         self._cache: Dict[str, Any] = {"id": pk}
 
+    @classmethod
+    def filter(cls, **filters: Dict[str, Any]) -> List["Model"]:
+        """Return a list of all objects that fit criteria."""
+        # Make sure attributes have 'id' in the first place.
+        attributes = filters.pop("__fields", None)
+        attributes = attributes or cls._filter_response_fields
+        attributes = ["id"] + [
+            attribute for attribute in attributes if attribute != "id"
+        ]
+
+        objects = communicator.filter_objects(
+            cls._app_name, cls._model_name, filters, attributes
+        )
+        models = []
+        for entry in objects:
+            model = cls(entry[0])
+            for field_name, value in zip(attributes[1:], entry[1:]):
+                field = model.fields[field_name]
+                # pylint: disable=protected-access
```

Comment:
We don't use pylint anymore.

Reply:
I have removed all the references to pylint.